pygame_functions.py

```diff
@@ -111,10 +117,12 @@ def setColour(self, colour):
 
 
 class newSprite(pygame.sprite.Sprite):
-    def __init__(self, filename, frames=1):
+    def __init__(self, filename, frames=1, altDims = None):
         pygame.sprite.Sprite.__init__(self)
         self.images = []
         img = loadImage(filename)
+        if altDims:
+            img = pygame.transform.scale(img, (altDims[0]*frames, altDims[1]))
```

Comment:
Resizes the image so all sprites will be the same as altDims. Should only be used when the scale factor is either an integer or it can perfectly size down the image, otherwise you'll probably make it look weird.